packages/foreman/foreman/foreman.spec

```diff
@@ -759,6 +762,33 @@ install -Dp -m0644 %{SOURCE5} %{buildroot}%{_tmpfilesdir}/%{name}.conf
 install -Dp -m0644 extras/systemd/%{name}.service %{buildroot}%{_unitdir}/%{name}.service
 install -Dp -m0644 extras/systemd/%{name}.socket %{buildroot}%{_unitdir}/%{name}.socket
 
+# SELinux libexec wrappers
+%if 0%{?scl:1}
+  cat > %{buildroot}%{_libexecdir}/%{name}/ruby-selinux <<-EOF
+  #!/bin/bash
+  # Shell wrapper with SELinux transition into foreman_rails_t domain.
+  source scl_source enable %{scl_prefix}
+  exec ruby "\$@"
+EOF
+  cat > %{buildroot}%{_libexecdir}/%{name}/sidekiq-selinux <<-EOF
+  #!/bin/bash
```

Comment:
I'm getting

```
systemd[1]: Started Foreman jobs daemon - orchestrator on sidekiq.
systemd[2852]: Failed at step EXEC spawning /usr/libexec/foreman/sidekiq-selinux: Exec format error
systemd[1]: dynflow-sidekiq@orchestrator.service: main process exited, code=exited, status=203/EXEC
systemd[1]: Unit dynflow-sidekiq@orchestrator.service entered failed state.
systemd[1]: dynflow-sidekiq@orchestrator.service failed.
```

removing the leading whitespace from the shebang fixes it

Reply:
Well I have learned a thing today. I tought `<<-EOF` removes intendation from the whole string. It does in fact only from the EOF line, and also these must be tabs not spaces.

Reply:
TIL :)

Reply:
It does [here](https://github.com/ekohl/ownca/blob/2034411c5819ac62a41bcc27ffe82f1e81ac9573/ownca#L5-L78). Tabs vs spaces perhaps?